src/api/test/functional/attributes_test.rb

```diff
@@ -131,6 +126,10 @@ def test_create_type
     assert_response :success
     get "/attribute/TEST/Dummy/_meta"
     assert_response :success
+    login_Iggy
+    delete "/attribute/TEST/Dummy/_meta"
+    assert_response 403
+    login_adrian
```

Comment:
@adrian why this is 403 I don't care. I do care that it is 403, that is all that matters. We don't verify code-flow but application behavior...

Please do not test for error messages, it only makes test suite maintenance harder...